tests/percy/lib/util.js

```diff
@@ -15,6 +15,16 @@ exports.unmasquerade = async function(page, HOME_PAGE) {
   await page.goto(`${HOME_PAGE}${link}`);
 }
 
+exports.removeAlert = async function(url, HOME_PAGE, page) {
```

Comment:
An optional improvement is to move "page.goto" outside "removeAlert" to make the intention clear and the code first visits a page and then removes alert from that page. After that, "removeAlert" will only need one parameter "page". This is along the the UNIX way of "small and focused components" :)

Reply:
Makes sense, changed removeAlert and added the goto back to anonymous.